server/crashmanager/Bugtracker/BugzillaProvider.py

```diff
@@ -302,14 +307,17 @@ def handlePOSTCreate(self, request, crashEntry):
         # Create the bug
         bz = BugzillaREST(self.hostname, username, password, api_key)
         ret = bz.createBug(**args)
-        if "id" not in ret:
-            raise RuntimeError("Failed to create bug: %s", ret)
+        if ret.status_code != 200:
+            content = json.loads(ret.content)
+            raise BugzillaProviderException(content['message'])
```

Comment:
We should probably try to handle more error cases here:

1) The resulting JSON might miss the `message`.
2) `ret.content` might not even parse as JSON (if the server has a hickup and returns nothing or e.g. the hardhat page)

In both cases, this would cause a 500 again. Instead we should probably wrap these cases in a `BugzillaProviderException` as well.

Reply:
Done